common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -180,7 +367,7 @@ def filter_templates(cls, template, course):
         return 'latex' not in template['template_id'] or course.use_latex_compiler
 
     def get_context(self):
-        _context = RawDescriptor.get_context(self)
+        _context = EditingMixin.get_context(self)
```

Comment:
This came from `RawDescriptor > XMLEditingDescriptor > EditingDescriptor`. `EditingDescriptor` methods have been moved to `EditingMixin` (that `ProblemBlock` inherits from now instead) so we can call it directly.